### ISSUE TITLE
Metrics appear disabled on remote node

### ISSUE THREAD

**Issue by mcrumm** [phoenixframework/plds#4]
Hi @philss! I just started `plds` and already it is amazing 🤩 great work!!

I noticed something odd– I have a Phoenix app generated from 1.6.0.rc-1, and from _its dashboard_ the Metrics tab is enabled.

If I start the app server like so:

```sh
$ elixir --name foo@localhost -S mix phx.server
```

and then I start plds (latest from github):

```sh
$ plds server --connect foo@localhost --open
```

When I select the `foo@localhost` node from the plds instance, the Metrics link is disabled. 

Curiously, looking at the dashboard from the _app instance_, I can see both nodes in the drop-down, and selecting either node shows the app instance's metrics.

Let me know if I can provide any more information and/or any help debugging!

**Comment by josevalim** [comment]
This is expected, `plds` does not have access to the metrics of the other node. @mcrumm do you see it disabled (i.e. with an `enable` button in it) or it does not show up at all?

@philss I think we should add a feature to the dashboard where we can set metrics and request_logger to false, so they don't show up at all, if we don't support this yet. Cause it will never work on plds, so we might as well disable it.

**Comment by philss** [comment]
Hi @mcrumm! I'm glad that you like it, and thanks! :D :purple_heart:

I couldn't find a way to make metrics work because we need to configure the metrics module in the router.
Maybe there is a way we can fetch that module from the other nodes similar to what we do with auto discovery for Ecto and Broadway. But I'm not sure how complex this is, and if works at all.

@josevalim good idea. I can tackle that later. Should I open an issue there?

**Comment by josevalim** [comment]
No need to open up an issue. We can track it here. :)

**Comment by josevalim** [comment]
@philss also, please try connecting to a node without `:os_mon` (it can be `iex --sname foo`) and see what happens. Those are our only three special pages.

**Comment by philss** [comment]
@josevalim it appears disabled:
![node-without-os_mon](https://user-images.githubusercontent.com/381213/134556838-2d846958-6a0d-4ea8-8969-4a81ee5cdc91.png)

**Comment by josevalim** [comment]
@philss cool. So that's good and correct. We just need to take care of request_logger and metrics then.

**Comment by mcrumm** [comment]
@josevalim @philss I am glad to know it's expected behaviour :) To be clear, the Metrics link appears disabled (with the enable button in it)

<img width="669" alt="Screen Shot 2021-09-23 at 1 07 28 PM" src="https://user-images.githubusercontent.com/168677/134576436-30e7ce97-66a5-442b-b3f3-3a35b060d0db.png">

**Comment by josevalim** [comment]
Yeah, desired but confusing behaviour. We should definitely improve the dashboard so we can disable them. :D
